budget/__main__.py

```diff
@@ -70,19 +92,21 @@ def validate(transaction, categories):
 
 # parses transaction fields & inserts a date field (today) if not specified
 def parseTransaction(params):
+    dateAdded = False;
     transaction = [e.strip() for e in params.split(',')]
     if len(transaction) is NUM_TRANSACTION_FIELDS - 1:
         print("Only 3 fields were specified. Assigning today to date field.")
         #transaction.insert(0, str(datetime.now())[:10])
```

Comment:
Looks like this line can be deleted.

Reply:
I don't understand why this line can be deleted. I don't think Python considers undeclared variables to be falsy but rather throw an error at run time.

Reply:
Sorry, my misunderstanding. I'll do that.